src/site/content/en/blog/learn-to-bundle/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Learn to bundle
+subhead: |
+    Learn to bundle your website.
+date: 2019-11-11
+hero: hero.png
+alt: A logo of a web bundle
+authors:
+  - kenjibaheux
```

Comment:
I think you'll need to add kenjibaheux to the contributors.json for the article to compile.